ShakeGuard.js

```diff
@@ -110,7 +110,34 @@ async function init(){
       password: "",
       multipleStatements: true
     });
-    
+
+    // Step 1: Create Table of Users and query it
+    const ShakeGuardUser = `CREATE DATABASE IF NOT EXISTS ShakeGuard;
```

Comment:
To clarify, we probably won't be using this code to start the backend.

We're using MongoDB, so this DB init code won't carry over.